nextcloud/core/templates/login.php

```diff
@@ -55,7 +55,7 @@
 		</p>
 
 		<div id="submit-wrapper">
-			<input type="submit" id="submit" class="login primary" title="" value="<?php p($l->t('Log in')); ?>" disabled="disabled" />
+			<input type="submit" id="submit" class="login primary" title="" value="<?php p($l->t('Log in with CSC Account')); ?>" disabled="disabled" />
```

Comment:
I wonder if this change was done in correct place. As maybe the correct change would have been into those i10n files?

Reply:
although while doing the change like this, one can be sure that only this location was impacted.

Reply:
Yes, only English is used for the login page, so the changes are appropriate.